dishes/butter-chicken.html

```diff
@@ -0,0 +1,69 @@
+    <img src="https://cafedelites.com/wp-content/uploads/2019/01/Butter-Chicken-IMAGE-64.jpg" width="200px"
+        height="auto" alt="image for Butter Chicken">
+    <p>
+        Butter chicken, also known as murgh makhani, is a popular Indian dish made with boneless chicken pieces that are
+        marinated in a mixture of yogurt, spices, and lemon juice, then grilled or baked until tender. <br>The chicken
+        is then simmered in a creamy tomato-based sauce that is flavored with butter, cream, and a blend of aromatic
+        spices such as garam masala, cumin, coriander, and turmeric.
+    </p>
+    <h3>Ingredients</h3>
+    <ul>
+        <li>Boneless chicken (breast or thigh meat)</li>
+        <li>Yogurt</li>
+        <li>Lemon juice</li>
+        <li>Ginger paste, Garlic paste</li>
+        <li>Garam masala, Cumin powder, Coriander powder, Turmeric powder, Red chili powder</li>
+        <li>Salt</li>
+        <li>Butter</li>
+        <li>Tomatoes</li>
+        <li>Heavy cream</li>
+        <li>Honey or sugar</li>
+        <li>Kasuri methi</li>
+    </ul>
+    <h3>Steps</h3>
+    <ol>
+        <li>
+            Marinate the chicken: In a large bowl, mix together the chicken, yogurt, lemon juice, ginger paste, garlic
+            paste, garam masala, cumin powder, coriander powder, turmeric powder, red chili powder, and salt. Cover and
+            refrigerate for at least 30 minutes or up to 24 hours.
+        </li>
+        <li>
+            Grill or bake the chicken: Preheat the grill or oven to 400°F (200°C). Thread the marinated chicken onto
+            skewers
+            (if grilling) or place the chicken on a baking sheet (if baking). Grill or bake the chicken until it is
+            cooked
+            through and lightly charred, about 10-15 minutes.
+        </li>
+        <li>
+            Make the sauce: In a large pot or pan, melt butter over medium heat. Add ginger paste, garlic paste, garam
+            masala, cumin powder, coriander powder, turmeric powder, and red chili powder. Cook for a minute until
+            fragrant.
+            Add chopped tomatoes and cook until the tomatoes are soft and mushy.
+        </li>
+        <li>
+            Puree the sauce: Using an immersion blender or transferring the mixture to a blender, puree the sauce until
+            it is smooth and creamy.
+        </li>
+        <li>
+            Finish the sauce: Return the pureed sauce to the pot or pan over medium heat. Add heavy cream, honey or
+            sugar,
+            and kasuri methi. Cook for a few minutes until the sauce thickens and the flavors meld together.
+        </li>
+        <li>
+            Add the cooked chicken: Add the grilled or baked chicken to the sauce and simmer for a few minutes until the
+            chicken is coated in the sauce and heated through.
+        </li>
+        <li>Serve: Garnish with fresh cilantro and serve with naan bread or basmati rice.</li>
+    </ol>
+</body>
+</html>
```

Comment:
new lines are missing at the end of all files.

It's standard to have a blank LF/CRLF as a file terminator.

You should use `Prettier` extension that comes along with the `VSCode` editor & configure it to format files automatically on save.

It should save you from the hassle of manually making such changes as well as other formatting feedbacks pointed out by your mentor in this PR

Reply:
please explain where i have to put new line, i am not understanding

Reply:
When you will end up working on production-grade applications, linters associated with such applications will end up complaining for such seemingly trivial concerns. It is important therefore that you make it a habit to write clean code from the start. 

One [helpful ref.](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3) that gives a gist of why new lines are a must

Reply:
Pls make a habit of working hand-in-hand with helpful tools like [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) & [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). This is a must & should not need to be reminded repeatedly (you must have read about it in your [Onboarding handbook guide](https://github.com/jalantechnologies/handbook/blob/main/engineering/onboarding-fresher.md)).

Reply:
i tried to add extra lines at end but after pushing to git, it is ignoring extra lines at end of the file

Reply:
updated the code. Thank you for the suggestions